javascript/builtins/Intl/NumberFormat.json

```diff
@@ -760,35 +760,35 @@
                 "description": "<code>number</code> param string value is decimal (not <code>Number</code>)",
                 "support": {
                   "chrome": {
-                    "version_added": false
+                    "version_added": "106"
                   },
                   "chrome_android": "mirror",
                   "deno": {
                     "version_added": false
                   },
                   "edge": "mirror",
                   "firefox": {
-                    "version_added": false
+                    "version_added": "116"
                   },
                   "firefox_android": "mirror",
                   "ie": {
                     "version_added": false
                   },
                   "nodejs": {
-                    "version_added": false
+                    "version_added": "19.0.0"
                   },
                   "oculus": "mirror",
                   "opera": "mirror",
                   "opera_android": "mirror",
                   "safari": {
-                    "version_added": false
+                    "version_added": "15.4"
```

Comment:
I just tested on Safari 16.5.1 (18615.2.9.11.7), and it still does not accurately support string values:

`new Intl.NumberFormat(undefined, { maximumFractionDigits: 20 }).format('111222333444555666777888999.111222333444555666')`
Safari 16.5.1: `111,222,333,444,555,670,000,000,000`

Firefox 115.0: `111,222,333,444,555,670,000,000,000`
Firefox 116.0: `111,222,333,444,555,666,777,888,999.111222333444555666`
Chromium 115.0: `111,222,333,444,555,666,777,888,999.111222333444555666`